src/main/java/com/example/ludogoriesoft/lukeriaerpapi/services/EmailService.java

```diff
@@ -0,0 +1,32 @@
+package com.example.ludogoriesoft.lukeriaerpapi.services;
+
+import jakarta.mail.internet.MimeMessage;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.stereotype.Service;
+
+@Service
+public class EmailService {
+    private static final Logger logger = LogManager.getLogger(EmailService.class);
+
+    private final JavaMailSender mailSender;
+
+    public EmailService(JavaMailSender mailSender) {
+        this.mailSender = mailSender;
+    }
+    public void sendHtmlEmail(String toEmail, String subject, String body) {
+        try {
+            MimeMessage message = mailSender.createMimeMessage();
+            MimeMessageHelper helper = new MimeMessageHelper(message, true, "UTF-8");
+            helper.setFrom("berki07092@gmail.com");
+            helper.setTo(toEmail);
+            helper.setSubject(subject);
+            helper.setText(body, true);
+            mailSender.send(message);
+        } catch (Exception e) {
```

Comment:
Catching generic exceptions is a bad practice, I would change it with } catch (MessagingException e) {
        logger.error("Messaging error occurred while sending email", e);
        throw new CustomEmailException("Messaging error occurred", e);
    } catch (MailException e) {
        logger.error("Mail server error occurred", e);
        throw new CustomEmailException("Mail server error occurred", e);
    }